modules/ui/arrow-ui-data/src/test/kotlin/arrow/ui/SumTest.kt

```diff
@@ -30,7 +32,8 @@ import io.kotlintest.shouldBe
 class SumTest : UnitSpec() {
   init {
 
-    val cf = { x: Int -> Sum.left(Id.just(x), Id.just(x)) }
+    val cf1 = { x: Int -> Sum.left(Id.just(x), Id.just(x)) }
```

Comment:
All of this code below could be cleaned up with `GenK` and `EqK`. `Sum` here is also only tested for `Sum.left` so a good genK would generate both